Description: Drop the #if check for the declaration of hid_get_extents()
Author: Michael Bienia <geser@ubuntu.com>
Bug: https://bugs.launchpad.net/pcb/+bug/1002964
Forwarded: https://bugs.launchpad.net/pcb/+bug/1002964/+attachment/3158021/+files/drop_check_global_included.patch
Index: pcb-20110918/src/hid/hidint.h
===================================================================
--- pcb-20110918.orig/src/hid/hidint.h	2011-05-20 06:36:57.000000000 +0200
+++ pcb-20110918/src/hid/hidint.h	2012-05-20 16:03:09.000000000 +0200
@@ -64,12 +64,8 @@
 /* Returns a filename base that can be used to output the layer.  */
 const char *layer_type_to_file_name (int idx, int style);
 
-#ifdef __GLOBAL_INCLUDED__
-
 /* Convenience function that calls the expose callback for the item,
    and returns the extents of what was drawn.  */
 BoxType *hid_get_extents (void *item);
 
-#endif
-
 void derive_default_filename(const char *pcbfile, HID_Attribute *filename_attrib, const char *suffix, char **memory);
